pkgs/development/web/kcgi/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkgconfig, fetchFromGitHub, libbsd }:
+
+stdenv.mkDerivation rec {
+  pname = "kcgi";
+  version = "0.10.5";
+  underscoreVersion = stdenv.lib.replaceChars ["."] ["_"] version;
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "kristapsdz";
+    repo = pname;
+    rev = "VERSION_${underscoreVersion}";
+    sha256 = "0ksdjqibkj7h1a99i84i6y0949c0vwx789q0sslzdkkgqvjnw3xw";
+  };
+  patchPhase = ''substituteInPlace configure \
+    --replace /usr/local /
+  '';
+  
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ ] ++ stdenv.lib.optionals stdenv.isLinux [ libbsd ] ;
+
+  configurePhase = "./configure";
```

Comment:
This is not necessary. Please use dontAddPrefix instead.

Reply:
I was unaware of that. Fixed.